packages/firebase_ml_model_downloader/firebase_ml_model_downloader/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: firebase_ml_model_downloader
 description: A Flutter plugin allowing you to use Firebase Ml Model Downloader.
 version: 0.1.0+13
-homepage: https://firebase.flutter.dev/docs/ml/overview
+homepage: https://firebase.google.com/docs/ml
```

Comment:
Should link the `firebase_ml_model_downloader` to this package?

Reply:
Hey @kevinthecheung, do you have a sense of where the ml_downloader plugin homepage ought to point to?

Reply:
Maybe https://firebase.google.com/docs/ml/flutter/use-custom-models is a better page

Reply:
Yes, I agree.

Reply:
```suggestion
homepage: https://firebase.google.com/docs/ml/flutter/use-custom-models
```